tests/e2e/README.md

```diff
@@ -16,6 +16,9 @@ Demo:
 ## Requirements
 A recent version of Fedora or CentOS9 with the following packages: bash, selinux and podman. 
 Make sure to enable `cgroupv2`.
+Enable the following on host machine before installing podman
+[hirte_repo](https://github.com/containers/qm/blob/main/tests/e2e/ContainerFile.control#L44-L45)
+
```

Comment:
As your patch will automatically handle that, no need to write here IMHO.

Reply:
@Yarboa we got another user sharing the iptables issue, cannot reproduce yet. Mergning this one, the next one we can fix the README. Thanks!

Reply:
@Yarboa your patch should fix it right? So no worries.